Gopkg.toml

```diff
@@ -1,26 +1,86 @@
+# Gopkg.toml example
```

Comment:
Is there any need to leave this example here as it is?

Reply:
It's autogenerated file from [dep](https://github.com/golang/dep)

With his help we can remove `vendors/` from repo =)